lib/swagger/fileLoader.ts

```diff
@@ -79,11 +79,15 @@ export class FileLoader implements Loader<string> {
   }
 
   public isUnderFileRoot(filePath: string) {
-    if (this.opts.fileRoot === undefined || this.opts.fileRoot === null) {
-      return false;
+    if (!this.opts.fileRoot) {
```

Comment:
I think line#82-#84 is not needed. No matter opts.fileRoot is provided or not, we need to check if the path is under root.

Reply:
if this.opts.fileRoot's type is string | undefined. If fileRoot is undefined, the path is under root or not?

Reply:
one question. In origin code, if opts.fileRoot is undefined, it return true. But now, it always retrun false when opts.fileRoot is undefined. Is it correct ?

Reply:
ok and it can be merged now @raych1